### DISCUSSION TITLE
I have an error when start using Github Copilot with VS Code in WSL. "certificate is not yet valid"

### DISCUSSION THREAD

**Discussion by huyhoangttql** [community/community#112148] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Detail error: [error] [default] Extension activation failed: "certificate is not yet valid".
Can someone please help me? Many thanks!

**Comment by pailoro** [comment]
Same here

**Reply by huyhoangttql** [reply]
Hi @pailoro, please check the solution from @DeandrePetersen below.

**Reply by pailoro** [reply]
@huyhoangttql  I've fixed mine syncing the time on my wsl. On terminal run `sudo ntpdate pool.ntp.br`. If you are in another country you should use another address

**Comment by DeandrePetersen** [comment, marked as answer]
Ensure that the system date and time on your machine are correctly set. If the system clock is incorrect, SSL certificates may appear as not yet valid or expired. Correcting the date/time settings should resolve the issue in many cases. If the system date/time is correct, check the expiration date of the SSL certificate associated with the server you're trying to connect to. If the certificate has expired, it will need to be renewed or replaced. Ensure that the SSL certificate presented by the server is properly signed by a trusted Certificate Authority (CA) and that the entire certificate chain is valid. Sometimes, an incomplete or misconfigured certificate chain can cause validation errors.

**Reply by huyhoangttql** [reply]
Thank you so much! It's working now
